Source/ASTextNode.mm

```diff
@@ -974,7 +974,7 @@ - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *)event
   BOOL linkCrossesVisibleRange = (lastCharIndex > range.location) && (lastCharIndex < NSMaxRange(range) - 1);
 
   if (inAdditionalTruncationMessage) {
-    return YES;
+    return !_passthroughNonlinkTouches;
```

Comment:
Right above here we short-circuit `if (!_passthroughNonlinkTouches)`. So this is equivalent to `return NO.` We should ideally remove this branch entirely, and simply return YES if we're in a link and NO otherwise. Whether we're in the truncation message or not doesn't matter for this method.

Reply:
@Adlai-Holler Agree and addressed